grpc/handlers_test.go

```diff
@@ -94,20 +94,22 @@ func TestHandlers_Tx(t *testing.T) {
 	})
 	cur, err := stream.RecvToClient()
 	require.NoError(t, err)
+	cID := cur.CursorId
```

Comment:
Is this change necessary?

Reply:
Yeah, response to Op_GET doesnt have a CursorID set. So it messes up the next Op_CLOSE call.